pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -182,17 +182,39 @@ func needsDocker(is []cloudprovider.InstanceType) bool {
 	return false
 }
 
-func (p *LaunchTemplateProvider) createLaunchTemplate(ctx context.Context, options *launchTemplateOptions) (*ec2.LaunchTemplate, error) {
-	output, err := p.ec2api.CreateLaunchTemplateWithContext(ctx, &ec2.CreateLaunchTemplateInput{
-		LaunchTemplateName: aws.String(launchTemplateName(options)),
-		LaunchTemplateData: &ec2.RequestLaunchTemplateData{
-			BlockDeviceMappings: []*ec2.LaunchTemplateBlockDeviceMappingRequest{{
+func (p *LaunchTemplateProvider) createLaunchTemplate(ctx context.Context, constraints *v1alpha1.Constraints, options *launchTemplateOptions) (*ec2.LaunchTemplate, error) {
+	blockDeviceMappings := []*ec2.LaunchTemplateBlockDeviceMappingRequest{{
```

Comment:
Maybe worth factoring out into a method, which will make things easier when we expose this to users.

Reply:
planning on refactoring when we support the provisioner root volume configuration